### ISSUE TITLE
Results appear after a long lag

### ISSUE THREAD

**Issue by mathetos** [jchristopher/searchwp-live-ajax-search#38]
I'm working locally, but have experienced close to a 10 second delay until any results appear. Here's the general environment:

1. Using SearchWP
2. Supplemental Search of a custom post type with almost all content in custom fields (ACF)
3. There are only 70 pages total
4. The form markup is pretty much identical to [this](https://searchwp.com/docs/kb/supplemental-search-step-by-step/)

If the immediate results can't be made quicker, can results at least be cached so appear quicker for future searches? Particularly very common searches (since you have that data available in SearchWP)?

**Comment by jchristopher** [comment]
Are you seeing the same sort of delays when utilizing only the supplemental search (e.g. not via Live Search but instead on a test page)?

The searches should be getting cached internally, but Live Search makes use of core Ajax calls (which a lot of other plugins (mis)use) so I'm wondering if you have also tried with only SearchWP and Live Search active, all other plugins inactive?

**Comment by mathetos** [comment]
No, the normal supplemental search goes to the correct page with instant results.

On my desktop after disabling all plugins I did see a marked improvement, but then after I activated them back the improvement was still there. So perhaps it's basically resolved. The extreme slowness was on my laptop, so I'll test there again with plugins disabled and see if I can find a clear "silver bullet" for which plugin really slowed it down.

Thanks!
